WebExtension/manifest.json

```diff
@@ -1,21 +1,27 @@
 {
-   "content_scripts": [ {
-      "all_frames": true,
-      "css": [ "xkit.css" ],
-      "exclude_matches": [ "https://www.tumblr.com/upload/image*, http://www.tumblr.com/upload/image*" ],
-      "js": [ "bridge.js", "jquery.js", "tiptip.js", "moment.js", "nano.js", "xkit.js" ],
-      "matches": [ "http://*.tumblr.com/*", "https://*.tumblr.com/*" ]
-   } ],
-   "description": "A fork of XKit, the extension framework for Tumblr.",
-   "homepage_url": "https://github.com/new-xkit/XKit",
-   "icons": {
-      "128": "icon.png"
-   },
-   "manifest_version": 2,
-   "minimum_chrome_version": "44.0",
-   "name": "New XKit",
-   "author": "New XKit Team",
-   "permissions": ["*://*/*", "*://*/", "unlimitedStorage", "storage", "http://*.tumblr.com/", "https://*.tumblr.com/" ],
-   "version": "7.7.1",
-   "web_accessible_resources": [ "manifest.json", "editor.js" ]
+  "content_scripts": [ {
+    "all_frames": true,
+    "run_at": "document_end",
+    "css": [ "xkit.css" ],
+    "exclude_matches": [ "https://www.tumblr.com/upload/image*, http://www.tumblr.com/upload/image*" ],
+    "js": [ "bridge.js", "jquery.js", "tiptip.js", "moment.js", "nano.js", "xkit.js" ],
+    "matches": [ "http://*.tumblr.com/*", "https://*.tumblr.com/*" ]
+  } ],
+  "description": "A fork of XKit, the extension framework for Tumblr.",
+  "homepage_url": "https://github.com/new-xkit/XKit",
+  "icons": {
+    "128": "icon.png"
+  },
+  "manifest_version": 2,
+  "minimum_chrome_version": "44.0",
+  "name": "New XKit",
+  "author": "New XKit Team",
+  "permissions": ["*://*/*", "*://*/", "unlimitedStorage", "storage", "http://*.tumblr.com/", "https://*.tumblr.com/" ],
+  "version": "7.8.0",
```

Comment:
also has version bump?

Reply:
thanks for pointing this out! I've reverted it to 7.7.1 for now